openedx/core/djangoapps/programs/utils.py

```diff
@@ -327,6 +328,11 @@ def supplement_program_data(program_data, user):
         program_data (dict): Representation of a program.
         user (User): The user whose enrollments to inspect.
     """
+    for organization in program_data['organizations']:
+        org_obj = get_organization_by_short_name(organization['key'])
```

Comment:
@rlucioni Are you talking about caching for the life time of the request? Or caching for the life time of the server? Or for some pre-determined time period like hours? Where is an example I can reference myself to?